packages/image_picker/image_picker/ios/Classes/FLTImagePickerPlugin.m

```diff
@@ -202,19 +224,19 @@ - (void)handleMethodCall:(FlutterMethodCall *)call result:(FlutterResult)result
   }
 }
 
-- (void)showCamera {
+- (void)showCameraWithImagePickerController:(UIImagePickerController *)imagePickerController {
```

Comment:
Optional nit: you could make these slightly less verbose by making them `withImagePicker:`; there's not non-controller image picker (AFAIK) to confuse it with.

Reply:
Fixed.